### PR TITLE
chore: add config files from mdn/content

### LINKED ISSUE #8618: Content suggestion: Adding a Configuration File
## What is the new suggestion?

I want the current warehouse to sync upstream, adding `.editorconfig` and `.prettierrc.json`.

---

我希望当前仓库向上游同步，添加 `.editorconfig` 和 `.prettierrc.json`。

## Why is it important or useful?
  
In this way, when translating content, it can be more relaxed.  Now I always need to manually adjust the indentation of the sample code (default: 4; original: 2) .

---

这样在翻译内容时，可以更轻松一些。现在的我老是需要手动调整示例代码的缩进（默认为 4，原文为 2）。

### PR DESCRIPTION
**Pull request by nschonni**
Closes #8618

Copied the package.json from the Markdownlint PR without any scripts as a baseline, along with the Prettier and Editorconfig from upstream

### CODE REVIEW

**Review by SphinxKnight — APPROVED** [review]
Thanks @nschonni for propagating this here🙇

File: package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "url": "git+https://github.com/mdn/translated-content.git"
+  },
+  "author": "MDN Web Docs",
+  "license": "SEE LICENSE IN LICENSE.md",
+  "bugs": {
+    "url": "https://github.com/mdn/translated-content/issues"
+  },
+  "homepage": "https://github.com/mdn/translated-content#readme",
+  "devDependencies": {
+  }
```
Comment (caugner) [inline package.json:22]: Could we reorder the fields, loosely based on the TOC of the [npm docs](https://docs.npmjs.com/cli/v8/configuring-npm/package-json)?

- name
- version
- description
- private
- homepage
- bugs
- license
- author
- repository
- scripts
- dependencies
- devDependencies
- engines

**Review by caugner — DISMISSED** [review]
LGTM, just the order of the `package.json` fields looked odd to me.

File: package.json (hunk 1 above)
Comment (nschonni) [reply package.json]: Done, but I didn't add the dependicies stub, since I'm not sure it will be used here

**Comment by nschonni** [comment]
@caugner I added in Dependabot, even though there are no packages yet.
I wasn't sure if the CODEOWNERS needed to be updated to match upstream with @schalkneethling as the assigned person to leverage the automerge job

**Comment by nschonni** [comment]
@bsmth not sure if you can land this

**Comment by bsmth** [comment]
> @bsmth not sure if you can land this

taking a look 👀

**Review by bsmth — APPROVED** [review]
Looks good to me 👍🏻

**Pull request merged**